src/common/components/repository-row/index.js

```diff
@@ -1,46 +1,46 @@
 import React, { PropTypes } from 'react';
 
-import Button from '../vanilla/button';
-
-import styles from './repositoryRow.css';
+import { TR, TD } from '../vanilla/table';
+import { parseGitHubRepoUrl } from '../../helpers/github-url';
 
 const RepositoryRow = (props) => {
 
   const {
-    errorMsg,
-    repository,
-    buttonLabel,
-    buttonDisabled,
-    onButtonClick
+    snap
   } = props;
 
+  const { fullName } = parseGitHubRepoUrl(snap.git_repository_url);
+
+  return (
+    <TR>
+      <TD><a href={ `/${fullName}/builds` }>{ fullName }</a></TD>
+      <TD></TD>
```

Comment:
There's an `/api/github/snapcraft-yaml/:owner/:name` route that you should use for this.

To the best of my knowledge, the GitHub API only lets us query this for a single repository, so we'll have to do this with one request per repository, perhaps in batches with `Promise.all`.  We should make the backend route cache positive responses in memcached (I'll try to do that today if I have time, but no promises).